app/components/views/GetStartedPage/SetMixedAcctPage/SetMixedAcctPage.jsx

```diff
@@ -116,32 +116,20 @@ export default ({ onSendBack, onSendContinue }) => {
                   </div>
                   <div className={classNames("is-column", styles.buttons)}>
                     <div className={classNames("is-row", styles.checkboxRow)}>
-                      <input
+                      <Checkbox
                         id={"mixed"+acctIdx}
-                        name={acctIdx}
-                        type="checkbox"
+                        label={<T id="getstarted.setAccount.mix" m="Set Mixed Account" />}
                         checked={mixedAcctIdx === acctIdx}
                         onChange={() => onSetMixedAcct(acctIdx)}
-                        value={acctIdx}
                       />
-                      <label htmlFor={"mixed"+acctIdx} className={styles.checkboxLabel}></label>
-                      <div className={styles.label}>
-                        <T id="getstarted.setAccount.mix" m="Set Mixed Account" />
-                      </div>
                     </div>
                     <div className={classNames("is-row", styles.checkboxRow)}>
-                      <input
+                      <Checkbox
                         id={"change"+acctIdx}
-                        name={"a"+acctIdx}
-                        type="checkbox"
+                        label={<T id="getstarted.setAccount.change" m="Set Unmixed Account" />}
                         checked={changeAcctIdx === acctIdx}
                         onChange={() => onSubmitSetChange(acctIdx)}
-                        value={acctIdx}
                       />
-                      <label htmlFor={"change"+acctIdx} className={styles.checkboxLabel}></label>
-                      <div className={styles.label}>
```

Comment:
Please ditch `.label` from .css

Reply:
Thank you for the review @amassarwi 
I tried to ditch `.label` as well as the `checkboxLabel`, but I did not find it in SetMixedAcctPage.module.css  :) Could you please help me find it?

Reply:
oh i see, then probably nothing to do here